client/src/pages/Login.js

```diff
@@ -0,0 +1,123 @@
+import React, { useState, } from 'react';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import Snackbar from '@material-ui/core/Snackbar';
+import SubmitButton from '../components/SubmitButton'
+import CustomTextField from '../components/CustomTextField'
+import axios from 'axios'
+import { storeUser } from '../utils/localStorage'
+
+
+const useStyles = makeStyles((theme) => ({
+  paper: {
+    marginTop: theme.spacing(8),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  form: {
+    width: '100%',
+    marginTop: theme.spacing(1),
+  },
+  mainContainer: {
+    border: '1px solid white',
+    borderRadius: '5px',
+    boxShadow: '0 2px 4px 0 gray',
+    backgroundColor: 'white',
+    marginTop: '100px',
+    width: '900px',
+  },
+}));
+
+export default function Login(props) {
+  const classes = useStyles();
+
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+  const [open, setOpen] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('')
+
+  const login = async (event) => {
+    try {
+      event.preventDefault()
+      let passedFields = true
+      if (email === '' || password === '') {
+        setErrorMessage('Please fill in all fields')
+        passedFields = false
+      }
+
+      if (!validateEmail(email)) {
+        setErrorMessage('Please enter your email')
+        passedFields = false
+      }
+      if (password.length < 7) {
+        setErrorMessage('Please enter your password')
```

Comment:
I think "Please enter a password with at least 7 characters." could be clearer.

Reply:
will do~